pkg/web/templates/partials/account/account_list.html

```diff
@@ -1,7 +1,7 @@
 {{ $canEditAccounts := not .IsViewOnly }}
 {{- with .AccountsList -}}
-<div>
-  <table class="table overflow-x-auto">
+<div class="table-container">
```

Comment:
The style added to fix the transaction list table is now applied here after realizing the overflow class wasn't working as expected.